astropy/table/__init__.py

```diff
@@ -39,3 +39,8 @@ class Conf(_config.ConfigNamespace):
 from ..io.misc import connect
 from ..io.votable import connect
 from . import jsviewer
+
+from ..io import registry
+
```

Comment:
Would it be an idea to comment here that the readers/writers register automatically, and this completes it by updating the documentation?